### ISSUE TITLE
Maintenance: { parser: "babylon" } is deprecated.

### ISSUE THREAD

**Issue by fbartho** [apollographql/eslint-plugin-graphql#222]
I enabled this plugin today, and when I run eslint, I get the following warning on the console:
```
{ parser: "babylon" } is deprecated; we now treat it as { parser: "babel" }.
```

This warning does not occur if this plugin is not in use.
